src/templates/preview.js

```diff
@@ -139,24 +140,27 @@ const JournalPage = ({
         value,
       })
     } />
-    <H4 css={css({color: 'black.1'})}>
-    {getHeading({
-          isFirstPage,
-          currentPage,
-          totalPages,
-          type,
-          value,
-        })}</H4>
-    <Box marginTop={6} width={[1, 2/3]}>
+    <H4 color='black.1'>
+      {getHeading({
+        isFirstPage,
+        currentPage,
+        totalPages,
+        type,
+        value,
+      })}
+    </H4>
+    <Box marginTop={6} width={['100%', 2/3]}>
     {blogs.map((blog,i) => (
       <div key={blog.childMdx.frontmatter.title}>
         <Blog frontmatter={blog.childMdx.frontmatter} />
-        { isLast(blogs, i)? '': 
-        <HorizontalRule
-          width={1}
-          borderWidth={1}
-          borderColor={'black.3'}
-        />
+        { isLast(blogs, i)
+          ? ''
+          : (
+            <Box
+              height={'1px'}
+              bg={'black.3'}
+            />
+          )
         }
       </div>
     ))}
```

Comment:
1. Mobile View is not responsive.
2. Headers are not looking as intended 

Seems styles are missing in `/blog` Page

Reply:
yes, blog pages are loading from @codebrahma/gatsby-theme-blog package, theme is hardcoded in that package. I am working on it